regression-test/suites/nereids_function_p0/load.groovy

```diff
@@ -27,6 +27,109 @@ suite("load") {
         DROP TABLE IF EXISTS `fn_test_bitmap`
     """
 
+    // test ipv4/ipv6
+    sql """ drop table if exists fn_test_ip_nullable """
+    sql """ CREATE TABLE IF NOT EXISTS fn_test_ip_nullable (id int, ip4 ipv4, ip6 ipv6) engine=olap
+                                                                                         DISTRIBUTED BY HASH(`id`) BUCKETS 4
+                                                                                         properties("replication_num" = "1") """
+    // make some special ip address
+    /***
+     回环地址
+    1;127.0.0.1;::1
+    // 私有地址
+    - 网络地址 (最小地址)
+    2;10.0.0.0;fc00::
+    - 广播地址 (最大地址)
+    3;10.255.255.255;fdff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
+    - 网络地址 (最小地址)
+    4;172.16.0.0;fc00::
+    - 广播地址 (最大地址)
+    5;172.31.255.255;febf:ffff:ffff:ffff:ffff:ffff:ffff:ffff
+    - 网络地址 (最小地址)
+    6;192.168.0.0;fe80::
+    - 广播地址 (最大地址)
+    7;192.168.255.255;ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
+    // 链路本地地址
+    8;169.254.0.0;fe80::
+    // 公有地址
+    9;8.8.8.8;2001:4860:4860::8888  // Google Public DNS
+    10;1.1.1.1;2606:4700:4700::1111  // Cloudflare DNS
+    // 组播地址
+    11;224.0.0.0;ff01::  // 所有主机
+    12;239.255.255.255;ff02::1  // 所有路由器
+    // 仅用于文档示例的地址
+    13;192.0.2.0;2001:0db8:85a3::8a2e:0370:7334
+    14;203.0.113.0;2001:db8::1
+    15;198.51.100.0;2001:db8::2
+    // 本地回环地址
+    16;localhost;::1
+    // IPv4 特殊地址
+    17;240.0.0.0;null  // 保留地址
+    18;255.255.255.255;null  // 广播地址
+    // 唯一本地地址
+    19;null;fd00::  // 唯一本地地址 (ULA)
+    // A 类地址
+    - 网络地址 (最小地址)
+    20;0.0.0.0;null
+    - 最大地址
+    21;127.255.255.255;null
+    // B 类地址
+    - 网络地址 (最小地址)
+    22;128.0.0.0;null
+    - 最大地址
+    23;191.255.255.255;null
+    // C 类地址
+    - 网络地址 (最小地址)
+    24;192.0.0.0;null
+    - 最大地址
+    25;223.255.255.255;null
+    // D 类地址
+    - 组播地址 (最小地址)
+    26;224.0.0.0;ff01::
+    - 最大地址
+    27;239.255.255.255;ff02::1
+    // 无效的多播地址
+    28;null;ff00::  // 保留地址
+    ***/
+
+    streamLoad {
+        table "fn_test_ip_nullable"
+        db "regression_test_nereids_function_p0"
+        file "fn_test_ip_special.csv"
+        set 'column_separator', ';'
```

Comment:
Why use seperator ';' ?

Reply:
my csv is separated by ';'
I think it doesn't matter for column_separator?